genon/preprocessor/facade/convert_processor.py

```
@@ -132,38 +132,106 @@
+            env.setdefault("LC_ALL", "C.UTF-8")
+
+            cmd = [
+                binary,
+                "-i", str(in_path),
+                "-o", str(out_dir),
+                "-t", tmp,
+                "-f", fonts_dir,
+                "-e", "-1",
+                "-p", "1",
+            ]
+            proc = subprocess.run(cmd, env=env, capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)